app/views/home/api_v1.html.erb

```diff
@@ -196,6 +196,48 @@
   <div class="col-md-12">
     <table class="table table-bordered api">
       <tr>
+        <td class="GET def">GET</td>
+        <td class="method">Key</td>
+        <td class="code">/api/v1/projects/XX/key/</td>
+        <td class="args">contribution_key(req)</td>
+      </tr>
+      <tr class="more" style="">
```

Comment:
Good question. It is there for every "more" class on this page. It doesn't seem to change when I remove it. Should I remove it everywhere it exists on this page?